PythonTool/WindowSettings.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+from visit import *
+
+
+def WindowSettings(Shading=False):
```

Comment:
A lot of these settings look hard-coded. Is there a way for the user to pass these in?